### ISSUE TITLE
Idea: Upgradable actors

### ISSUE THREAD

**Issue by raulk** [filecoin-project/ref-fvm#717]

**Comment by Stebalien** [comment]
No, assuming we can reach consensus on the current actor addressing proposals.

**Comment by maciejwitowski** [comment]
> No, assuming we can reach consensus on the current actor addressing proposals.
@Stebalien does it mean we don't need this now? Or we'll need it later?
